recipes/sundials/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cc -I${PREFIX}/include -L${PREFIX}/lib -o cvRoberts_dns cvRoberts_dns.c -lm -lsundials_cvode -lopenblas -lsundials_nvecserial
```

Comment:
Well, this test obviously not have complete coverage, but it proved better than `EXAMPLES_ENABLE=1`.